core-processor/src/configs.rs

```diff
@@ -56,7 +56,7 @@ pub struct AllocationsConfig {
 impl Default for AllocationsConfig {
     fn default() -> Self {
         Self {
-            max_pages: WasmPageNumber(code::MAX_WASM_PAGE_COUNT),
+            max_pages: code::MAX_WASM_PAGE_COUNT.into(),
```

Comment:
nooooooo, oh my god, it is completely wrong idea

Reply:
Why?

Reply:
Because WasmPageNumber is struct which reporesents Wasm Pages and  MAX_WASM_PAGE_COUNT is our runtime characteristic which is not connected with wasm pages preferences